src/web/overlays/styles/_variables.scss

```diff
@@ -12,6 +12,7 @@ $darkblue: #110e1a;
 $purple: #7d00c9;
 
 $white: #ffffff;
+$opaquewhite: rgba(255, 255, 255, .25);
```

Comment:
This could use a better description... something like $whiteAlpha-25 (white with an alpha of .25)